package.json

```diff
@@ -126,8 +126,8 @@
     "copy-webpack-plugin": "6.4.1",
     "css-loader": "6.7.1",
     "css-minimizer-webpack-plugin": "3.4.1",
-    "electron": "18.0.0",
-    "electron-builder": "23.3.3",
+    "electron": "18.3.15",
+    "electron-builder": "22.10.3",
```

Comment:
just want to double check that this version downgrade is intentional.

Reply:
Yeah this is intentional🥲 npm audit recommended the fix and that did fix 6 high vulnerabilities. Would you suggest to revert the downgrade?

Reply:
As long as it is intentional, let's make the change. Security is always a priority.